src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -962,6 +962,29 @@ void uct_rc_mlx5_iface_common_dm_cleanup(uct_rc_mlx5_iface_common_t *iface)
 #endif
 }
 
+#if HAVE_DECL_MLX5DV_CREATE_QP
+void uct_rc_mlx5_common_set_qp_dv_attr(uct_rc_mlx5_iface_common_t *iface,
+                                       struct mlx5dv_qp_init_attr *dv_attr,
+                                       unsigned scat2cqe_dir_mask)
+{
+#if HAVE_DECL_MLX5DV_QP_CREATE_ALLOW_SCATTER_TO_CQE
+    unsigned dir;
+
+    dv_attr->comp_mask |= MLX5DV_QP_INIT_ATTR_MASK_QP_CREATE_FLAGS;
+    ucs_for_each_bit(dir, scat2cqe_dir_mask) {
+        ucs_assert(dir < UCT_IB_DIR_NUM);
+        if (iface->super.super.config.max_inl_cqe[dir] == 0) {
+            dv_attr->create_flags = MLX5DV_QP_CREATE_DISABLE_SCATTER_TO_CQE;
```

Comment:
AFAIU MLX5DV_QP_CREATE_DISABLE_SCATTER_TO_CQE we need to add only if dir == TX, return regardless

Reply:
doesn't it affect both directions?
e.g if max_inl_cqe[RX]==0, how can we disable scatter2cqe for receiver?

Reply:
isn't it default, i.e. nothing to pass?